R/measures.R

```diff
@@ -47,16 +49,26 @@
 #'
 #' @examples
 #' \dontrun{
-#' riem_measures(station = "VOHY", date_start = "2000-01-01", date_end = "2016-04-22")
+#' riem_measures(station = "VOHY", date_start = "2000-01-01", date_end = "2016-04-22", trace = 1e-4, missing = "empty")
```

Comment:
I think it'd make sense to choose either 1e-4 or .0001 as notation in the whole file.